{"Timestamp":"2023-11-11T19:33:08.1982187+04:00","Level":"Error","MessageTemplate":"An unhandled exception has occurred while executing the request.","Exception":"System.Exception: error here\r\n   at PersonReplations.Application.Features.PersonFeatures.AddPersonRequestHandler.Handle(AddPersonRequest request, CancellationToken cancellationToken) in C:\\Users\\George\\source\\repos\\PersonRelations.API\\PersonReplations.Application\\Features\\PersonFeatures\\AddPersonRequest.cs:line 41\r\n   at MediatR.Wrappers.RequestHandlerWrapperImpl`1.<>c__DisplayClass1_0.<<Handle>g__Handler|0>d.MoveNext()\r\n--- End of stack trace from previous location ---\r\n   at PersonRelations.API.Controllers.PersonController.Add(AddPersonRequest request) in C:\\Users\\George\\source\\repos\\PersonRelations.API\\PersonRelations.API\\Controllers\\PersonController.cs:line 20\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","Properties":{"EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HMV2PCP6ISH6:00000003","RequestPath":"/api/Person/Add","ConnectionId":"0HMV2PCP6ISH6"}}
{"Timestamp":"2023-11-11T19:33:53.5892551+04:00","Level":"Fatal","MessageTemplate":"Application startup exception","Exception":"System.InvalidOperationException: Unable to resolve service for type 'System.Diagnostics.Stopwatch' while attempting to activate 'PersonRelations.API.Middlewares.ErrorHandling'.\r\n   at Microsoft.Extensions.Internal.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)\r\n   at Microsoft.Extensions.Internal.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)\r\n   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass5_0.<UseMiddleware>b__0(RequestDelegate next)\r\n   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()\r\n   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.<ConfigureApplication>b__27_0(RequestDelegate next)\r\n   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()\r\n   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)","Properties":{"EventId":{"Id":6},"SourceContext":"Microsoft.AspNetCore.Hosting.Diagnostics"}}
{"Timestamp":"2023-11-11T19:36:10.7916393+04:00","Level":"Error","MessageTemplate":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000}","Exception":"System.Exception: error here\r\n   at PersonReplations.Application.Features.PersonFeatures.AddPersonRequestHandler.Handle(AddPersonRequest request, CancellationToken cancellationToken) in C:\\Users\\George\\source\\repos\\PersonRelations.API\\PersonReplations.Application\\Features\\PersonFeatures\\AddPersonRequest.cs:line 41\r\n   at MediatR.Wrappers.RequestHandlerWrapperImpl`1.<>c__DisplayClass1_0.<<Handle>g__Handler|0>d.MoveNext()\r\n--- End of stack trace from previous location ---\r\n   at PersonRelations.API.Controllers.PersonController.Add(AddPersonRequest request) in C:\\Users\\George\\source\\repos\\PersonRelations.API\\PersonRelations.API\\Controllers\\PersonController.cs:line 20\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at PersonRelations.API.Middlewares.ErrorHandling.InvokeAsync(HttpContext context) in C:\\Users\\George\\source\\repos\\PersonRelations.API\\PersonRelations.API\\Middlewares\\ErrorHandling.cs:line 22","Properties":{"RequestMethod":"POST","RequestPath":"/api/Person/Add","StatusCode":"{ Message = error here }","Elapsed":"(500) Internal Server Error","__4":786,"ActionId":"694372cf-66a4-42e4-be09-ea6caafcfe03","ActionName":"PersonRelations.API.Controllers.PersonController.Add (PersonRelations.API)","RequestId":"0HMV2PFED4IAO:00000001","ConnectionId":"0HMV2PFED4IAO"},"Renderings":{"Elapsed":[{"Format":"0.0000","Rendering":"(500) Internal Server Error"}]}}
